index.js

```diff
@@ -0,0 +1,156 @@
+//task №1
+
+const arrayNumbers = [1 , 2, 3, 4, 5, 6, 7, 8];
+
+let convertNumbersInArray = (arrayNumbers) => {
+	return arrayNumbers.map((number) => (number % 2 === 0) ? number += 1 : number -= 1);
+}
+
+console.log(convertNumbersInArray(arrayNumbers));
+
+
+//task №2
+
+const arrayStrings = ['hello', 'wOrLd'];
+let convertStringsInArray = (arrayStrings) => {
+	return arrayStrings.map((string) => string[0].toUpperCase() + string.slice(1).toLowerCase());
+}
+
+console.log(convertStringsInArray(arrayStrings));
+
+
+//task №3
+
+const usersList = [
+	{
+		id: 1,
+		name: 'Alex',
+	},
+	{
+		id: 2,
+		name: 'Toma',
+	},
+	{
+		id: 3,
+		name: 'Max',
+	},
+]
+
+let showUsers = (usersList, id) => {
+	return usersList.filter((user) => {
+		return !id.includes(user.id);
+	});
+}
+
+console.log(showUsers(usersList, [1]));
+
+
+//task №4
+
+let forCheckPalindrome = (value) => value.split('').reverse().join('') === value;
+
+console.log(forCheckPalindrome('112211'));
+
+
+//task №5
+const users = [
+	{
+		id: 1,
+		first_name: 'Jeanette',
+		last_name: 'Penddreth',
+		email: 'jpenddreth0@census.gov',
+		gender: 'Female',
+		ip_address: '26.58.193.2',
+	},
+	{
+		id: 2,
+		first_name: 'Petr',
+		last_name: 'Jackson',
+		email: 'gfrediani1@senate.gov',
+		gender: 'Male',
+		ip_address: '229.179.4.212',
+	},
+];
+
+let convertUsersNewArray = (users) => {
+	return users.map((user) => {
+		return {
```

Comment:
same as above. If you are not planning to create any stuff inside the arrow function body, don't use `return` key word